src/php/please.ts

```diff
@@ -0,0 +1,52 @@
+const completionSpec: Fig.Spec = {
+  name: "please",
+  description: "Statamic Please command",
+  generateSpec: async (tokens, executeShellCommand) => {
+    var out = await executeShellCommand("php please list --format=json");
```

Comment:
Could you replace this with a `const`?
```suggestion
    const out = await executeShellCommand("php please list --format=json");
```